package.json

```diff
@@ -64,9 +64,11 @@
     "jest": "^28.1.3",
     "lint-staged": "^13.0.2",
     "mini-css-extract-plugin": "^2.6.0",
+    "node-sass": "^7.0.3",
```

Comment:
The [webpack documentation](https://webpack.js.org/loaders/sass-loader/) suggests using `dart-sass` instead of `node-sass`. In our case it should be much of a problem - although if you run into any - please reconsider using the recommendation

Reply:
As per the advice I change the node-sass or dart-sass (up to date name - sass)